### ISSUE TITLE
Docusaurus & shopifytech.tips

### ISSUE THREAD

**Issue by math-ecen** [math-ecen/shopify-recipes#4]
To make this repo more accessible to non-developers or more technical merchants and store staff who might want to use the information in this repo without necessarily having the technical knowledge or development environment handy to do so, the repo will also have a website analog: **[Shopify Tech Tips](https://shopifytech.tips/)**. 

Shopify Tech Tips will be a Docusaurs site created to host a blog for miscellaneous posts and tutorials as well as structured documentation for the examples in the repo and more formal tutorials on how to use the Shopify tech stack to do a range of things. 

Let's consider three of Shopify's best written resources: 

- The **[Shopify Help Docs](https://help.shopify.com/)** are a broad user manual for Shopify with a mixture of text, screenshots, and videos which educate a merchant at beginner through to power user functionality within the core platform.
- The **[Shopify Developer docs](https://shopify.dev/docs)** are an in depth documentation of specific API and technical information which a developer can put to work in their code. 
- The **[Shopify Blog](https://www.shopify.com/blog)** is a written part of Shopify's content strategy which provides light touch education on a myriad of topics that act as case studies and inspiration for aspirational merchants. 

Each resource is great at what it does. The first two segment less technical and non-technical users from developers and offer each a reference on how to use the tools within their grasp to use Shopify. The last resource is an excellent one for Shopify's content marketing strategy but also awesome in that it brings the focus back to businesses that are using Shopify in practice. 

Where Shopify Tech Tips could be powerful is in extending and linking to these resources but approaching it from the point of view of a potential or current power user. Someone who can articulate what they want, and have attempted as a power user to make their vision happen. 

In summary, the new site should focus on being practical, focused on adding value to businesses, and offering context for power users on how Shopify underlying tech works which as a developer we can get through the dev docs but isn't approachable for power users and non-technical mechants. 

We can get functionality that is low hanging fruit from apps in the **[Shopify App Store](http://apps.shopify.com/)**, update and extend the merchant-facing **[unsupported tutorials](https://community.shopify.com/c/Shopify-Design/Unsupported-Theme-Tutorials-Cart-Forms-Variants-and-more/td-p/638735)**, and simplify developer oriented tutorials to be more power-user approachable.
